app/page/AboutPage.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import DefaultNavigation from '../component/navigation/DefaultNavigation';
+import Config from '../config';
+import Link from 'react-router/lib/Link';
+import { FormattedMessage } from 'react-intl';
+import connectToStores from 'fluxible-addons-react/connectToStores';
+
+class AboutPage extends React.Component {
+
+  componentDidMount() {
+    return null;
+  }
+
+  render() {
+    const about = Config.aboutThisService[this.props.currentLanguage];
+    return (
+      <DefaultNavigation
+        className="front-page fullscreen"
+        disableBackButton
+        showDisruptionInfo
+        title={Config.title}
+        showLogo={Config.useNavigationLogo}
+      >
+        <div className="page-frame fullscreen momentum-scroll">
+          <h1 id="about-header">
+            <FormattedMessage
+              id="about-this-service" defaultMessage="About this service"
+            />
+          </h1>
+          <p dangerouslySetInnerHTML={{ __html: about.about }} />
```

Comment:
This is seriously uncool from the security standpoint as it allows for potential Client-side XSS. The same goes for the desktop wrapper text.